documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -112,7 +114,11 @@ You can migrate virtual machines to {virt} from the command line.
 
 [IMPORTANT]
 ====
-You must ensure that all xref:prerequisites[prerequisites] are met.
+*  You must be logged in as a user with the {ocp} `cluster-admin` privileges.
+
+*  VMware only: You must have the xref:vmware-privileges_{context}[VMware privileges].
+
+*  You must ensure that all xref:prerequisites[prerequisites] are met.
```

Comment:
@sbeskin-redhat Why is it necessary to say this? Doesn't this go without saying?

Reply:
@stoobie Just to remind the user of the prerequisites and to give a link to them before he starts the actual procedure of migration.